__tests__/app.test.js

```diff
@@ -112,8 +112,45 @@ describe("GET /api/articles", () => {
       .get("/api/banana")
       .expect(404)
       .then(({ body }) => {
-        console.log(body);
         expect(body.msg).toBe("Incorrect endpoint");
       });
   });
 });
+
+describe("GET /api/articles/articles_id/comments", () => {
+  test("200: should respond with an array of aall comment from article with a given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const allComments = body.comments;
+        allComments.forEach((eachComment) => {
```

Comment:
As we are using assertions within a forEach we need to also assert in this test the correct length of the array. If the endpoint sends back an empty array (which is should not for this test) currently those asertions won't run and so the test will give a false positive. the length assertion will protect against this